shared_requirements.txt

```diff
@@ -68,3 +68,7 @@ yarl
 azure-identity
 openai
 dnspython
+promptflow-core
```

Comment:
Can we add some kind of inline comment to mark this section as provisional?

Reply:
Don't think it'll work unfortunately. I filed #37097